.github/workflows/ci.yml

```diff
@@ -0,0 +1,82 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    name: Console CI
+    runs-on: ubuntu-18.04
+    timeout-minutes: 5
+    steps:
+
+    - name: Set up Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.14
+      id: go
+
+    - name: Setup Go binary path
+      shell: bash
+      run: |
+        echo "::set-env name=GOPATH::${{ github.workspace }}"
+        echo "::add-path::${{ github.workspace }}/bin"
+
+    - name: Check out code
+      uses: actions/checkout@v2
+      with:
+        path: src/github.com/containerd/console
+        fetch-depth: 25
+
+    - name: Checkout project
+      uses: actions/checkout@v2
+      with:
+        repository: containerd/project
+        path: src/github.com/containerd/project
+
+    - name: Install dependencies
+      env:
+        GO111MODULE: off
+      run: |
+        go get -u github.com/vbatts/git-validation
+        go get -u github.com/kunalkushwaha/ltag
+        go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
+        (
+          cd "${GOPATH}/src/github.com/golangci/golangci-lint/cmd/golangci-lint"
+          git checkout v1.23.8
+          go build -v && go install
+        )
```

Comment:
Not for this PR, but we could make use of go modules for these;

1. change to a directory outside of the project (to prevent the go.mod and/or vendor directory from being updated)
2. run `go get "github.com/golangci/golangci-lint/cmd/golangci-lint@${GOLANG_CI_VERSION}"`


edit: although I see they don't recommend installing from source; https://golangci-lint.run/usage/install/#local-installation